### ISSUE TITLE
Make some tutorials to show how to use Project Sidewalk data

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2625]
While the new Project Sidewalk deployments have a number of built-in analytic and visualization tools like Label Map, Sidewalk Gallery, and hopefully more (with @manaswis' PhD work), we occasionally receive inquiries like the following or requests for different types of visualizations of our data.

![image](https://user-images.githubusercontent.com/1621749/124182130-4e015c80-da6b-11eb-8ca8-c24e7062dc6e.png)

So, I think it would be helpful to have some lil tutorials on how to use our API to create your own visualizations.  I can't believe I never thought of this... but most (popular) APIs have lil example demos showing how to use an API.

I could imagine doing this in our Wiki and linking to it from the API page. We could create multiple examples depending for various levels of technical complexity. The easiest would be to show how to use a tool like Tableau or carto.com to import and visualize our data. For example, @gari01234 used carto.com to visualize San Pedro data (see example visualizations https://github.com/LigaPeatonal/evaluando-spgg).

We've now been talking about this on Slack and @manaswis recommended https://kepler.gl/demo, which @misaugstad then used to quickly make this:

![image](https://user-images.githubusercontent.com/1621749/124182617-f0b9db00-da6b-11eb-9744-735d3a5b3e26.png)

This seems great to me!

And @manaswis, I think this also relates to your thesis work in that it: first, we have demand (from journalists particularly) to make their own visualizations of our data; and second, that showing ways to do this potentially broadens the each of our data and how it could make impact.

Exciting!

**Comment by manaswis** [comment]
Yes, its really exciting! For the last thesis piece, I had been thinking of exploring intersections with data journalism and working with data journalists for a while. However, won't have time to do so before graduation. Maybe post-PhD - its on my TODO list :)

**Comment by misaugstad** [comment]
Alright I threw together a tutorial using Kepler! Spent more time on it than I meant to, but it's probably worth it to make sure the guide doesn't leave out crucial info for a non-technical user.

https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Making-your-own-visualization-of-Project-Sidewalk-data

**Comment by jonfroehlich** [comment]
The new API pages start to address this but don't walk users through how build their own visualizations, which is what this ticket is about. So, I still think this could be useful...
